.github/workflows/continuous-integration.yaml

```diff
@@ -433,15 +433,62 @@ jobs:
         run: |
           make pcc
 
+      # Checked for changes from specific directories in order to determine. More specifically, 
+      # this is used in regular CIs to avoid launching Pytest, checking codeblocks, building docs 
+      # or other steps if the associated files were not touched. For most, we also check that the
+      # linux MD5 has not changed, which means that no libraries got updated. This is done in order
+      # to handle PRs which only upgrades dependencies
+      - name: Get all src, tests, docs and codeblock files that have changed from main
+        id: changed-files
+        if: ${{ env.IS_PR && steps.conformance.outcome == 'success' && !cancelled() }}
+        uses: tj-actions/changed-files@v37
+        with:
+          files_yaml: |
+            src:
+              - src/**
+              - deps_licenses/licenses_linux_user.txt.md5
+            tests:
+              - tests/**
+              - deps_licenses/licenses_linux_user.txt.md5
```

Comment:
strictly speaking, I would add `src/concrete/ml/pytest` in `tests`

Reply:
ah yes indeed !